about.html

```diff
@@ -0,0 +1,142 @@
+                    <em>Sawyer, E. K.</em>, and Sarko, D. K.  2017. Evolution of the somatosensory brainstem in mammals.  In Kaas, J. H.and Herculano-Houzel, S., Evolution of the Nervous System. 2:437-459 New York, NY: Elsevier</a>
+                  </li>
+                </ol>
+
+                Conference Abstracts
+                <ol>
+                  <li>Turner E.C., <em>Sawyer, E. K.</em>, Kaas J. H.  The visual system of the Northern Elephant Seal (Mirounga angustirostris).  Poster presentation at the Society for Neuroscience meeting; Nov. 12, 2016; San Diego, CA.</li>
+
+                  <li><em>Sawyer, E. K.</em>, Liao C.-C., Qi H., Balaram P., Matrov D., and Kaas J. H. Barreloid-like structures in the ventroposterior medial subnucleus of the somatosensory thalamus in prosimian galagos. Poster presentation at the Society for Neuroscience meeting; Nov. 15, 2014; Washington, DC.</li>
+
+                  <li>Balaram P., <em>Sawyer, E. K.</em>, Kaas, J. H. VGLUT1 and VGLUT2 identify driving (class 1) and modulatory (class 2) glutamatergic projections in the somatosensory system of prosimian, New World, and Old World primates.  Poster presentation at the Society for Neuroscience meeting; Nov. 15, 2014; Washington, DC.</li>
+
+                  <li>Sawyer E. K., Leitch D. B.,Catania K. C. “Organization of the spinal trigeminal nucleus in the Star-nosed mole.” Poster presentation at the Society for Neuroscience meeting; Oct. 16, 2012; New Orleans, LA.</li>
+
+                  <li><em>Sawyer, E. K.</em>, Leitch D. B. and Catania K. C. “Organization of the spinal trigeminal nucleus in the Star-nosed mole”. Poster session at the 2012 Neuroscience graduate program retreat 2012 Sept. 20; Nashville, TN.</li>
+
+                  <li>Leitch, D. B., <em>Sawyer, E. K.</em>, Catania K. C.  “Mechanoreception and the integumentary sensory organs in Crocodylia.” Presentation at annual meeting of the Society for Neuroscience meeting, Mechanisms of Somatosensation nanosymposium. Oct. 16, 2012; Ernest N Morial Convention Center, New Orleans, LA.  </li>
+
+                  <li><em>Sawyer, E. K.</em>., Clover L., Crow T. J., Hof P. R., Chance S. A. “Neuronal organization and plasticity in Wernicke’s area differ between humans and chimpanzees.”  Poster presentation at The Primate Mind meeting; Jun. 4-7, 2009; Erice, Italy.</li>
+                </ol>
+              </ul>
+            </section>
+
+
+          </main>
+
+
+          <footer>
+            <nav>
+              <ul>
+                <li>
+                  <a href = "index.html">Home</a>
+                </li>
+                <li>
+                  <a href = "portfolio.html">Portfolio</a>
+                </li>
+                <li>
+                  <a href = "code-journal.html">Code Journal</a>
+                </li>
+              </ul>
+            </nav>
+          </footer>
+
+
+        </body>
+        </html>
+      </body>
+      </html>
```

Comment:
These last two lines, closing the `body` and `html` tags are duplicated here, in `code-journal.thml`, and `portfolio.html`.